src/jquery.autocomplete.js

```diff
@@ -761,9 +761,12 @@
             }
 
             $.each(that.suggestions, function (i, suggestion) {
-                var foundMatch = suggestion.value.toLowerCase().indexOf(value) === 0;
-                if (foundMatch) {
-                    bestMatch = suggestion;
+                var foundMatch = false;
+                if (suggestion.value !== undefined) {
```

Comment:
Why would suggestion value be undefined?

Reply:
I don't know exactly, I didn't try to understand why, but it happens when hitting the backward key and causes the error.